dio/test/basic_test.dart

```diff
@@ -46,7 +45,7 @@ void main() {
       Dio().get('http://http.invalid'),
       throwsA((e) => e is DioError && e.error is SocketException),
     );
-  });
+  }, testOn: "vm");
```

Comment:
May fix next PR， to normalize the error between Default & Browser HttpAdapter

Reply:
That would be a great improvement for 5.0